src/views/Correction.vue

```diff
@@ -606,9 +606,7 @@ export default {
         }
       }
       if (setting === 'add' && !added) {
-        // TODO: remove hard-coded entity type when fee codes are available from Pay team
-        // this.filingData.push({ filingTypeCode: filing, entityType: this.entityType })
-        this.filingData.push({ filingTypeCode: filing, entityType: EntityTypes.BCOMP })
+        this.filingData.push({ filingTypeCode: filing, entityType: this.entityType })
```

Comment:
Nice!

Reply:
Yes, the Correction filing code is valid for both BCOMP and COOP (I checked).